manual/parts/cspace.tex

```diff
@@ -425,11 +427,12 @@ \subsection{Addressing Capabilities}
 
 \begin{description}
 \item[Cap A.] The first CNode has a 4-bit guard set to 0x0, and an
-  8-bit radix. Cap A resides in slot 0x60 so it may be referred to by
-  any address of the form 0x060xxxxx (where xxxxx is any number,
-  because the translation process terminates after translating the
-  first 12 bits of the address). For simplicity, we usually adopt the
-  address 0x06000000.
+  8-bit radix. Cap A resides in slot 0x60 so, provided that it is not
+  a \obj{CNode} capability, it may be referred to by any address of
+  the form 0x060\textit{aaaaa} (where \textit{aaaaa} is any sequence
```

Comment:
Maybe we should stick with using "xxxxx" instead of "aaaaa", because 'a' is a valid hex char and that could become confusing even if there is an explanation given immediately after. In the example below it could say then "adopt the address 0x06000000 instead of e.g. 0x060abcde"

Reply:
I agree that `a` isn't the best choice because it is a valid hexadecimal digit. As you noted below, `x` is confusing in one of the other cases because the `x`s directly follow the `0x` prefix. I tried italicizing the placeholder `x`s, but the typeset result looked bad. `n`, which you suggest below, seems like the best choice to me. I've changed each instance to use `n`.